src/main/java/frc/robot/commands/SwitchAngle.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.Launch;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+
+public class SwitchAngle extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final Launch m_subsystem;
+  private boolean isUsed;
+  private int timer;
+
+  /**
+   *
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public SwitchAngle(Launch subsystem) {
+    m_subsystem = subsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    isUsed = false;
+    timer = 0;
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if(m_subsystem.getAngle() && !isUsed){
+      m_subsystem.setFar();
+      isUsed = true;
```

Comment:
A couple suggestions for this.
 * isUsed isn't the best variable name... better would be isDone or didWork
 * if you put this in the initialize, then it will only get called once.
 * you could use a real timer rather than counting how many times you're called.  that'd be better.  There is example of a timer [here](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gettingstarted/Robot.java)